Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -374,10 +376,28 @@ protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
                                 } else if (VUNGLE_MREC == adConfig.getAdSize()) {
                                     vungleMrecAd = sVungleRouter.getVungleMrecAd(placementReferenceId, adConfig);
                                     if (vungleMrecAd != null) {
-                                        View adView = vungleMrecAd.renderNativeView();
+                                        final View adView = vungleMrecAd.renderNativeView();
                                         if (adView != null) {
                                             isLoadSuccess = true;
-                                            layout.addView(adView);
+                                            final int width = ViewUtility.dpToPixels(mContext, VUNGLE_MREC.getWidth());
+                                            final int height = ViewUtility.dpToPixels(mContext, VUNGLE_MREC.getHeight());
```

Comment:
not sure if we can use non-public com.vungle.warren.utility.ViewUtility api

Reply:
fixed.